cheep.py

```diff
@@ -0,0 +1,45 @@
+import boto3
+import sys
+
+def get_regions():
+    regions = ['us-west-1','us-west-2','us-east-1','us-east-2','ap-northeast-1','ap-northeast-2','ap-northeast-3','ap-south-1','ap-southeast-1','ap-southeast-2','ca-central-1','eu-central-1','eu-north-1','eu-west-1','eu-west-2','eu-west-3']#,'sa-east-1','us-east-1','us-east-2','us-west-1','us-west-2']
+
+    ec2_client = boto3.client('ec2', region_name='us-east-1')
```

Comment:
I'm guessing the region_name here doesn't matter and is just for the sake of pointing the client somewhere to find all the regions. Is that correct?

Reply:
yes, it's for the region of the endpoint for the API